docs/source/data_ingestion_and_preparation/running_a_data_preparation_job.md

```diff
@@ -1,55 +1,63 @@
 # Running Custom Data Preparation Code
 
-Once you have written your data preparation code, you can now use it to process data when loading it into Sycamore.
+If you have chosen to write you data preparation code with an editor, you can run it using the terminal in the Jupyter container or the Sycamore-Importer container. You can also write and run code using a [Jupyter notebook](using_Jupyter.md).
 
-## Using Jupyter container
+## Using the terminal in the Jupyter container
 
-The easiest way to use your data processing code is to use the Jupyter notebook shell in the Sycamore-Jupyter container. You enable this container when deploying Sycamore. [LINK]
+The easiest way to run your data preparation code is to use the Jupyter notebook shell in the Jupyter container, which is launched by default.
 
-Add your code to the bind-dir XXX NEED THIS INFO!
+1. Add your data preparation code to the bind directory so you can access it in the Jupyter container. On your local machine, add your files to:
 
-Go to your Jupyter notebook server and go to Jupyter shell
+```../jupyter/bind_dir```
 
-Run this command:
+2. In your browser, go to the URL for your [Jupyter container](using_jupyter.md) and create a new terminal. You can do this by clicking on "File" in the menu, going to "New," and then selecting "Terminal".
 
-XXXXXXXXXXXXX
+3. In the terminal, run this command:
 
-You can also choose to run your data processing code from the Docker terminal by:
+```python /bind-dir/your-file-name.py```
```

Comment:
python /app/work/bind_dir/your-file-name.py
or python bind_dir/your-file-name.py

since by default it dumps you in /app/work.

Reply:
Ack